src/internal/k8s/namespace.go

```diff
@@ -18,6 +18,15 @@ func GetNamespaces() (*corev1.NamespaceList, error) {
 	return clientset.CoreV1().Namespaces().List(context.TODO(), metaOptions)
 }
 
+func UpdateNamespace(namespace *corev1.Namespace) (*corev1.Namespace, error) {
+	message.Debugf("k8s.UpdateNamespace(%v)", namespace)
+
+	clientset := getClientset()
+	updateOptions := metav1.UpdateOptions{}
```

Comment:
What do you think about adding this to the function parameters? 

One of the comments I had in #480 was pointing out that a nice-to-have update in the future is to add the options to the `GetPods()` call so the tests can do more specific queries instead of having to use kubectl.

Not sure it makes too much sense for an update call but wanted to mention it anyways.

Reply:
I like it, but let's keep it as a separate PR because I'd also like to refactor all the k8s work to leverage Go generics too.